src/clj/rems/db/test_data.clj

```diff
@@ -23,8 +23,9 @@
 ;;; generate test data
 
 (defn- create-users-and-roles! [users attrs]
-  (doseq [attr (vals attrs)]
-    (create-user! attr))
+  (doseq [user (vals users)]
+    (when-let [data (get attrs user)]
+      (create-user! data)))
```

Comment:
one consequence of this change is that, for example, `+fake-user-data+` can contain entries that do not exist in `+fake-users+`. in the scope of this PR `elixir-alice` is a user that only needs to exist as token data because they are mapped into a existing user.

Reply:
We could also have one user that does not map to an existing user. Since that'll be the case often and we could make sure a new user is created fine.

Reply:
added extra user into `+fake-user-data+` that doesn't map into any existing user :+1: